palace/fem/multigrid.hpp

```diff
@@ -70,17 +67,15 @@ std::vector<std::unique_ptr<FECollection>> inline ConstructFECollections(
     }
   }
   std::reverse(fecs.begin(), fecs.end());
+
```

Comment:
I thought the rule was no white space unless comment comes after?

Reply:
Added this to match `ConstructFiniteElementSpaceHierarchy` and `ConstructAuxiliaryFiniteElementSpaceHierarchy`. I don't think there's a rule here, just what looks best (subjective probably).